### DISCUSSION TITLE
Progression Discord?

### DISCUSSION THREAD

**Discussion by MrJay1012** [onimp/oni_multiplayer#39] [General]
Hi! Tested this out with a friend, we LOVED it. The host material selector bug made it unplayable but aside from that we enjoyed it. I see it's in the planned fixes so no complaints there! However is there a discord to follow progress with little updates or anything. I understand it's a project being done in your own time and it wont be rushed, but would be nice to have a place for updates if and when they do come. Thank you <3

**Comment by zuev93** [comment]
Hi [MrJay1012](https://github.com/MrJay1012),

Thank you very much for the feedback and the ask!

So far we don't have any discord server or channel to discuss. We are still available for a chat via DM.

I've created a ticket to create discord (or something similar) #67.
